src/pages/index.js

```diff
@@ -1,94 +1,204 @@
-// ! ||--------------------------------------------------------------------------------||
-// ! ||                                    CONSTANTS                                   ||
-// ! ||--------------------------------------------------------------------------------||
+// CONSTANTS
 const addCardAddButton = document.querySelector(selectors.cardAddButton);
 const updateProfileButton = document.querySelector(selectors.profileEditButton);
 const cardFormElement = document.querySelector(selectors.cardPopup);
 const profileFormElement = document.querySelector(selectors.profilePopup);
+const avatarFormElement = document.querySelector(selectors.avatarPopup);
 const profileHeadingInput = profileFormElement.querySelector(selectors.profileName);
 const profileDescriptionInput = profileFormElement.querySelector(selectors.profileDescription);
+const avatarURLInput = avatarFormElement.querySelector(selectors.avatarURL); // Correct association with avatar form
 const newImagePopup = new PopupWithImage(selectors.cardImagePopup);
+const updateAvatarButton = document.querySelector(selectors.avatarEditButton);
 newImagePopup.setEventListeners();
+
 const editProfileFormValidator = new FormValidator(formValidationConfig, profileFormElement);
 const cardFormValidator = new FormValidator(formValidationConfig, cardFormElement);
-editProfileFormValidator.enableValidation();
-cardFormValidator.enableValidation();
-
-//INITIAL CARDS
-const cardSection = new Section(
-  { items: initialCards,
-    renderer: (item) => {
-      const card = renderCard(item);
-      cardSection.addItem(card);
-    }
+const updateAvatarFormValidator = new FormValidator(formValidationConfig, avatarFormElement);
+
+handleValidation(editProfileFormValidator);
+handleValidation(cardFormValidator);
+handleValidation(updateAvatarFormValidator);
+
+// API SETUP
+const api = new Api({
+  baseURL: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "06384f1a-a606-42fb-9776-b06e7d5ab968",
+    "Content-Type": "application/json",
   },
-  selectors.cardSection,
-);
-cardSection.renderItems();
-
-//NEW CARD
-const newCardPopup = new PopupWithForm(
-  selectors.cardPopup,
-  handleAddCardFormSubmit
-);
-newCardPopup.setEventListeners();
+});
 
-//USER INFO
+// USER INFO
 const userInfo = new UserInfo({
-    nameSelector: selectors.profileHeadingElement,
-    descriptionSelector: selectors.profileDescriptionElement
+  nameSelector: selectors.profileHeadingElement,
+  descriptionSelector: selectors.profileDescriptionElement,
+  avatarSelector: selectors.avatarImageElement,
 });
 
-//USER PROFILE
-const updateProfilePopup = new PopupWithForm(
-  selectors.profilePopup,
-  handleProfileSubmit
-);
+// INITIAL CARDS
+let cardSection = new Section({
+  items: [],
+  renderer: (item) => {
+    const card = renderCard(item);
+    cardSection.addItem(card);
+  },
+}, selectors.cardSection);
+
+api.initialPageLoad()
+  .then(([cards, user]) => {
+    cardSection.setItems(cards);
```

Comment:
You don't have `setItems` in `Section` that's why there is an error  and nothing works